include/zephyr/input/input_analog_axis_settings.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_SETTINGS_H_
+#define ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_SETTINGS_H_
+
+#include <stdint.h>
+#include <zephyr/device.h>
+
+/**
+ * @brief Analog axis API
+ * @defgroup input_analog_axis Analog axis API
+ * @ingroup io_interfaces
+ * @{
+ */
+
+/**
+ * @brief Save the calibration data.
+ *
+ * @param dev Analog axis device.
+ *
+ * @retval 0 If successful.
+ * @retval -errno In case of any other error.
+ */
+int analog_axis_calibration_save(const struct device *dev);
```

Comment:
I'd make it explicit here that this depends on the settings subsys (docs)

Reply:
good call, added